hack/after-build/update-generated-deep-copies.sh

```diff
@@ -62,6 +62,6 @@ function generate_deep_copies() {
 # Currently pkg/api/deep_copy_generated.go is generated by the new go2idl generator.
 # All others (mentioned above) are still generated by the old reflection-based generator.
 # TODO: Migrate these to the new generator.
-DEFAULT_VERSIONS="v1 authorization/__internal authorization/v1beta1 autoscaling/__internal autoscaling/v1 batch/__internal batch/v1 extensions/__internal extensions/v1beta1 componentconfig/__internal componentconfig/v1alpha1 metrics/__internal metrics/v1alpha1"
```

Comment:
Can we remove this file completely, and just add a line to https://github.com/kubernetes/kubernetes/blob/master/hack/after-build/run-codegen.sh?